views/entity.html

```diff
@@ -10,254 +10,233 @@
-                name: "public_register_name",
-                value: public_register_name
-            }) }}
+        {{ govukInput({
+          label: {
+            text: "Country"
+          },
+          classes: "govuk-!-width-two-thirds",
+          id: "serviceAddressCountry",
+          name: "serviceAddressCountry",
+          value: service_address.serviceAddressCountry if service_address
+        }) }}
 
-            {{ govukInput({
-                label: {
-                    text: "What is the registration number of the entity?",
-                    classes: "govuk-label--m",
-                    isPageHeading: false
-                },
-                id: "registration_number",
-                name: "registration_number",
-                value: registration_number
-            }) }}
+        {{ govukInput({
+          label: {
+            text: "Postcode or ZIP code (if applicable)"
+          },
+          classes: "govuk-input--width-10",
+          id: "serviceAddressPostcode",
+          name: "serviceAddressPostcode",
+          autocomplete: "postal-code",
+          value: service_address.serviceAddressPostcode if service_address
+        }) }}
 
-            <h2 class="govuk-heading-m">What information is shown on the online register</h2>
-            <p class="govuk-body">All details provided about the overseas entity will be available on the online register</p>
+        {% endcall %}
+      {% endset -%}
 
-            {% include "includes/continue-button.html" %}
-        </form>
-    </div>
+      {{ govukRadios({
+        idPrefix: "is_service_address_same_as_principal_address",
+        name: "is_service_address_same_as_principal_address",
+          fieldset: {
+            legend: {
+              text: "Is the correspondence address the same as the principal or registered office address?",
+              isPageHeading: false,
+              classes: "govuk-fieldset__legend--m"
+            }
+          },
+          items: [
+            {
+              value: 1,
+              text: "Yes",
+              checked: is_service_address_same_as_principal_address === 1
+            },
+            {
+              value: 0,
+              text: "No",
+              checked: is_service_address_same_as_principal_address === 0,
+              conditional: {
+                html: serviceAddressHtml
+              }
+            }
+         ]
+      }) }}
+
+      {{ govukInput({
+        label: {
+          text: "Which email address at the overseas entity should we send communications to?",
+          classes: "govuk-label--m",
+          isPageHeading: false
+        },
+        hint: {
+          text: "We’ll use this to send the Overseas Entity ID number, the notice of registration, and any other digital correspondence"
+        },
+        id: "email",
+        name: "email",
+        value: email
+      }) }}
+
+      {% set hintText = "For example, limited company" %}
```

Comment:
Screenshot shows this hint text as "For example, limited company or partnership"

Reply:
Interesting: Here https://roe-v5.herokuapp.com/register/entity we have "For example, limited company" . I was looking at this it seems to be different from the screenshot.

Reply:
I agree, probably best to take the text from the prototype as it is probably more up to date than the screenshot.